### PR TITLE
Added media queries to enable responsive desktop version

### PR DESCRIPTION
**Pull request by Koshao-sys**
In this pull request, I have used media queries to enable a responsive desktop version while maintaining the design on Figma.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Koshao-sys,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

❗️❗️You have submitted a project, that has no software license and license section on the Readme. Kindly check the the [license section](https://github.com/microverseinc/readme-template/blob/master/README.md#-license-) here to learn more.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -150,88 +155,105 @@ <h1 class="head-recent">My Recent <br> Works</h1>
...
+                <div class="about-img-left">
+                    <img src="./images/illustration-about-me-2-desktop.svg" alt="">
+                </div>
+                <div class="about-me">
+                    <h2>About me</h2>
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <button class="btn-resume">Get My Resume</button>
+                </div>
```
Comment (bobb-Rob) [inline index.html]: - [ ] Your website looks wonderful. Well done 👏 
But there is a little issue that I see with the website's responsiveness. The goal of this project is to make your project very responsive across all views, from mobile to tablet and various laptop screens. I recorded a video that explains my observations. Kindly fix this issue. Please [watch the video here.](https://drive.google.com/file/d/1LbTyyf5aZWPAenEyQQKtvEqJNgrclvTN/view)

- Essentially, your website's about section is not centrally aligned in views between ``768px - 1075px`. This means that your visitor that uses tablets and the likes will not have a good experience with you site. Please ensure your site looks good across all screen types.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Koshao-sys , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Koshao-sys** [comment]
Kindly note that i have managed to make the entire site responsive to different screen sizes which was failing in my previous submission. Thanks

**Review by mugishasam123 — APPROVED** [review]
Hi  @Koshao-sys  :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive PR , keep up the energy :+1: 
- NO linter errors :heavy_check_mark: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**